src/pages/filter/common/dashboard/components/body/widget-tab/components/widget-sub-tab/components/widget-sub-tab-enum.tsx

```diff
@@ -0,0 +1,57 @@
+import styles from '../widget-sub-tab.module.css'
+
+import { Fragment, ReactElement } from 'react'
+import { observer } from 'mobx-react-lite'
+
+import { t } from '@i18n'
+import dashboardStore from '@pages/filter/common/dashboard'
+import { MAX_SUB_TAB_ROWS_AMOUNT } from '@pages/filter/common/dashboard/dashboard.constants'
+import { IEnumPropertyStatus } from '@service-providers/common'
+
+interface IWidgetSubTabEnumProps {
+  unit: IEnumPropertyStatus
+  onSelectUnit: () => void
+}
+
+export const WidgetSubTabEnum = observer(
+  ({ unit, onSelectUnit }: IWidgetSubTabEnumProps): ReactElement => {
+    const variantsLeft = unit.variants
+      ? unit.variants.length - MAX_SUB_TAB_ROWS_AMOUNT
+      : null
+
+    const handleOpenEnumDialog = (variantName: string) => {
+      dashboardStore.setEnumVariant(variantName)
```

Comment:
This function causes a problem: 
1. Open the Dashboard for `xl_PF0005_wes` preset
2. Toggle a dropdown for `Inheritance` -> `Has_Variant`
3. 40 variants are show -> click the last one
4. You can see a popup with active `+ Add Attribute` button, but you don't see the chosen variant (it's on the 6th page). It can be confusing.

Also if you then click cancel button the chosen variant doesn't disappear: you can click the `Has_Variant` header and you'll see that `+ Add Attribute` button is still active. I guess it should disappear after cancelling